subiquity/server/server.py

```diff
@@ -272,6 +274,9 @@ class SubiquityServer(Application):
 
     supported_variants = ["server", "desktop"]
 
+    # Classes of exceptions to suppress creation of apport reports for
+    apport_suppressed_exceptions: tuple[Exception] = (AutoinstallError,)
```

Comment:
What's your thoughts of handling this through the inheritance chain?  So adding a `NonReportableException` or similar and have `AutoinstallError` inherit that and then we can just `isinstance(foo, NonReportableException)` ?

Reply:
I think it's better, but early on I wasn't sure about it yet because I was still playing with the API structure. I'm a little more confident in it now so I'll make the changes.